### PR TITLE
add more game crafts

### PR DESCRIPTION
**Pull request by wsor4035**
what the title says

### CODE REVIEW

File: crafts.lua [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+        sand = "fl_stone:sand",
+        gravel = "fl_topsoil:gravel",
+        steel_ingot = "fl_ores:iron_ingot",
+        gold_ingot = "fl_ores:gold_ingot",
+        mese_crystal_fragment = "fl_ores:iron_ingot",
+        torch = "fl_light_sources:torch",
+        diamond = "fl_ores:diamond",
+        clay_lump = "fl_bricks:clay_lump",
+        water_bucket = "fl_liquids:water_source",
+        empty_bucket = "air",
```
Comment (S-S-X) [inline crafts.lua]: Is this `air` really correct item here? Or should it be empty item / no replacements at all and such?

Comment (wsor4035) [reply crafts.lua]: https://github.com/mt-mods/basic_materials/pull/8/commits/ab90fdc6c38270438ceaf1b1ad2f223269ef4ea4 forgot to save, it was from when i was checking replacements and i3

**Review by S-S-X — APPROVED** [review]
seems fine but also did not actually test anything..

**Comment by MysticTempest** [comment]
PR is heavily based off my own work here: https://github.com/mt-mods/basic_materials/pull/7

**Comment by CartridgeDoesCode** [comment]
Seems cool i guess…..

**Pull request merged**